app/webpacker/components/RegistrationsV2/Register/CompetingStep.jsx

```diff
@@ -79,6 +80,10 @@ export default function CompetingStep({
 
   const [processing, setProcessing] = useState(false);
```

Comment:
What goes against just using the `irProcessing` that is passed in from the backend as initial value for this state?

Reply:
I tried to fix that with the useEffect, but it still occurs

Reply:
But the loop is being caused by the `useEffect`. My point was to remove the `useEffect` and _instead_ use the initialization value here.

Reply:
it is not. I had it exactly like you proposed previously and it caused the same loop

Reply:
https://github.com/thewca/worldcubeassociation.org/pull/10297/commits/fc0048e35ff3b8793a4c2af269d871cd93c011b4
see first commit